### DISCUSSION TITLE
What color do you get if you mix together the two colors that appear twice in the Google logo?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#359] [Galaxy Trivia] [answered]
**Hint:** *** Haze

**Comment by peterbaccaro** [comment, marked as answer]
The two colours that appear twice in the Google logo are blue and red. If you mix these colours together, you get the colour purple.

**Reply by ParanoidUser** [reply]
Well done! 👏👏 Congratulations on your well deserved [Galaxy Brain 🥉](https://github.com/peterbaccaro?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Silver for 16 answers), and don't hesitate to post your own questions to the community. 😊🎉

**Comment by thuongtruong1009** [comment]
The two colors that appear twice in the Google logo are blue and red. When you mix blue and red together, you get purple. However, it's worth noting that the exact shade of purple you get may depend on the specific shades of blue and red that are used in the Google logo.
